pytorch_blade/tests/quantization/__init__.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+from torch_blade.quantization import is_available as is_quantization_available
+from torch_blade.tensorrt import is_available as is_tensorrt_available
+from torch_blade.testing.common_utils import TestCase
+
+TORCH_VERSION = tuple(int(x) for x in torch.__version__.split(".")[:2])
+
+if TORCH_VERSION >= (1, 10):
+    zero_point_dtype = torch.int32
+else:
+    zero_point_dtype = torch.long
```

Comment:
use `torch_blade.utils.torch_version_number` and `torch_blade.utils.parse_version`

Reply:
done